modules/installation-gcp-managing-dns-solution.adoc

```diff
@@ -0,0 +1,53 @@
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-gcp-enabling-user-managed-DNS_{context}"]
+= Enabling user-managed DNS
+
+You can install a cluster with a DNS solution that you manage. As a result, you can manage the API and Ingress DNS records in your own system rather than managing the records in the cluster. For example, your organization's security policies might not allow the use of public DNS services such as Google Cloud DNS. In such scenarios, you can use your own DNS service to bypass the public DNS service. You can manage your DNS only for the IP addresses of the API and Ingress servers.
+
+[NOTE]
+====
+If you enable user-managed DNS during installation, the installation program will not provision DNS records for the API and Ingress servers. To ensure that you can log in to your cluster, you must provision the DNS records for the API and Ingress servers after installation.
+====
+
+.Prerequisites
+
+* You installed the `jq` package.
+* You installed the `gcloud` CLI tool.
+
+.Procedure
+. Use a text editor to edit the `install-config.yaml` file before you deploy your cluster and add the following stanza:
+.. To enable your own DNS for control plane machines, compute machines, or both:
++
+[source,yaml]
+----
+platform:
+  gcp:
+    userProvisionedDNS: Enabled <1>
+----
+<1> Enable DNS management.
+
+. Use the `gcloud` CLI to find the IP address of the API server and then provision the corresponding DNS record.
```

Comment:
This step along with the next step "_Use the gcloud CLI to find the IP address of the ingress service..._" should be moved to after `Deploying the cluster` and before `Logging in to the cluster by using the CLI`, rather than during `Creating the installation configuration file`.